.github/workflows/update-jib-base-image.yml

```diff
@@ -0,0 +1,66 @@
+name: Auto-Update Jib Base Image
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * 5" # Every Friday at Midnight
+
+jobs:
+  update_build_gradle:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Install GitHub CLI
+        run: |
+          sudo apt-get update
+          sudo apt-get install gh
+
+      - name: Update build.gradle and create PR
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          # Extract current version from JShellAPI build.gradle
+          gradle_file="JShellWrapper/build.gradle"
+          current_version=$(grep -oP "(?<=from\.image\s=\s'eclipse-temurin:)\d+" $gradle_file)
+          
+          if [ -z "$current_version" ]; then
+            echo "Failed to extract version from $gradle_file \"from.image\""
+            exit 1
+          fi
+          
+          # Fetch the latest eclipse-temurin image
+          latest_version=$(curl -s "https://hub.docker.com/v2/repositories/library/eclipse-temurin/tags/?page_size=100" | \
```

Comment:
https://hub.docker.com/v2/repositories/library/eclipse-temurin/tags?name=23-alpine

This could be used instead and just check if count field is 0 or not.

Reply:
Don't do this, if we're on Java 23 and some how Java 25 releases, you'll miss 25 and end up on 24. While, unrealistic, it's good to handle the edge cases.

Reply:
Alright